### ISSUE TITLE
Stuck at the Optimize Server section of First run

### ISSUE THREAD

**Issue by dkopyone** [awarre/Optimize-WsusServer#23]
I did a first Run on the script on our server 2019 WSUS installation.

I have gotten no errors.. but it has been sitting at the Optimize Server section with a blinking cursor for almost 96 hours. 

Can it take this long to complete the process?

Just to note, This WSUS server hasn't been touched for a few years.. No maintenance or anything.

Is there anything i should do? Or just let it go?

**Comment by viniperini** [comment]
Yeah, it can take a long time to complete the first run. its complete normal.
